### DISCUSSION TITLE
My one question.

### DISCUSSION THREAD

**Discussion by Amazinations** [keiwando/toonsquid#633] [Q&A]
Will there ever be ToonSquid for Android/Play Store?

**Comment by DWrightAnimation** [comment]
I wish I can answer that question, but I don't know the answer. For now, I'd suggest looking into Krita on Android/Play Store, it doesn't necessarily have vector graphics, but it is good for animation.

**Comment by Cat-Butler** [comment]
Currently, the closest you'll get to an [official answer is in the FAQs](https://toonsquid.com/handbook/guides/faq/#platforms).
